gensrc/thrift/FrontendService.thrift

```diff
@@ -1817,5 +1825,7 @@ service FrontendService {
     TTableReplicationResponse startTableReplication(1: TTableReplicationRequest request)
 
     TGetPartitionsMetaResponse getPartitionsMeta(1: TGetPartitionsMetaRequest request)
+
+    TReportCloudCompactionResponse reportCloudCompaction(1: TReportCloudCompactionRequest request)
```

Comment:
maybe stick to LakeCompaction name convention, since the agreement is yet to be reached for the *Cloud* prefix.